lms/static/js/Markdown.Converter.js

```diff
@@ -1285,27 +1285,12 @@ else
         //  attacklab: Utility functions
         //
 
-        var _problemUrlChars = /(?:["'*()[\]:]|~D)/g;

```

Comment:
@cahrens Because this method has been removed in the latest version of the Markdown Library. The changes I have done are from the latest version Markdown library; picking the snippet that was used to solve this issue.